### PR TITLE
Fixed static analyser warnings in sim-packages 4

### PR DESCRIPTION
**Pull request by civanch**
Code clean-up:
1. Fixed static analyser warnings; removed empty lines; removed obsolete commented lines.
2. Removed few obsolete Geant4 disclaimers.
3. Substitute link of vdt by vdt_header.
4. A part of cout is substituted by LogInfo.

Part of fixed code is used in production but this PR should not change any result.

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by mmusich** [comment]
+1
minor changes in `Alignment/LaserAlignmentSimulation/plugins/LaserAlignmentProducer.h` moving to MT friendly module type.

File: Alignment/CocoaModel/interface/ALIUnitsTable.h [hunk 1]
```diff
@@ -1,24 +1,5 @@
...
-// * regarding  this  software system or assume any liability for its *
-// * use.                                                             *
-// *                                                                  *
-// * This  code  implementation is the  intellectual property  of the *
-// * GEANT4 collaboration.                                            *
-// * By copying,  distributing  or modifying the Program (or any work *
-// * based  on  the Program)  you indicate  your  acceptance of  this *
-// * statement, and all its terms.                                    *
-// ********************************************************************
+// This class was derived from Geant4 by Pedro Arce
```
Comment (davidlange6) [inline Alignment/CocoaModel/interface/ALIUnitsTable.h]: @civanch - I don't find this reasonable - this class is copied from Geant4 as far as I see (and its code still in the g4 codebase)

Comment (civanch) [inline Alignment/CocoaModel/interface/ALIUnitsTable.h]: @davidlange6 , this license is a bit obsolete - Geant4 License/Disclaimer text is now different. I would guess, that this copy was a wrong choice from the beginning but I am not sure we should improve this sub-library.

Comment (civanch) [inline Alignment/CocoaModel/interface/ALIUnitsTable.h]: @davidlange6 , if you cannot merge this I will return back the disclaimer today evening.

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by civanch** [comment]
@mmusich , please resign - the concern of David has been addressed in the new commit.

**Comment by davidlange6** [comment]
@civanch @mmusich  - I looked for a moment at how we use this ALIUnitsTable - fortunately it has no users. Unfortunately there is also a CocoaUnitsTable - though it also may be unused. Perhaps these can be deleted (there is a stray include or two)

**Comment by civanch** [comment]
@davidlange6 , it is used in one line of code inside Alignment/CocoaModel/src/ParameterMgr.cc , unfortunately, this use is not marginal and cannot be removed in an easy way, because the last class is used everywhere in the package. Of course, I am not sure that this package is not obsolete.

**Comment by davidlange6** [comment]
ah thanks - seems like there are a few good choices - either, use G4 directly (we depend on g4 after all) - maybe this has a downside with respect to dependencies/big libraries; or write the few lines needed to replace the ALIUnitDefinition::GetValueOf(unitstr)...

On Jul 18, 2016, at 11:59 AM, Vladimir Ivantchenko <notifications@github.com<mailto:notifications@github.com>> wrote:


@davidlange6<https://github.com/davidlange6> , it is used in one line of code inside Alignment/CocoaModel/src/ParameterMgr.cc<http://parametermgr.cc> , unfortunately, this use is not marginal and cannot be removed in an easy way, because the last class is used everywhere in the package. Of course, I am not sure that this package is not obsolete.

**Comment by civanch** [comment]
@cmsbuild , it seems several tests (not only this) are stacked for a long time.

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
@mmusich , please resign

**Comment by mmusich** [comment]
+1

**Comment by mmusich** [comment]
@civanch @davidlange6, to answer your previous messages, I am under the impression that the whole package `/Alignment/CocoaModel/` is somehow either not used or not updated since long time. I think we should survey the AlCa contact for its usage. @ghellwig

**Comment by ghellwig** [comment]
@mmusich in another PR (don't remember which one) this question was asked already and you wanted to contact privately the only customer of this package that you are aware of (you said he is not on github). Any outcome from this?

**Comment by davidlange6** [comment]
+1

**Pull request merged**